src/test/kotlin/calculator/OperatorTest.kt

```diff
@@ -0,0 +1,36 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+import org.junit.jupiter.params.provider.ValueSource
+
+internal class OperatorTest {
+
+    @ParameterizedTest(name = "연산자: {0}")
+    @ValueSource(strings = ["+", "-", "-", "-"])
+    fun `문자열로 연산자 찾기`(symbol: String) {
+        assertThat(Operator.of(symbol)).isIn(*Operator.values())
+    }
+
+    @ParameterizedTest(name = "연산자: {0}")
+    @ValueSource(strings = ["^", " ", "1", "a"])
+    fun `지원하지 않는 연산자 예외`(symbol: String) {
+        assertThatThrownBy {
+            Operator.of(symbol)
+        }.isInstanceOf(IllegalArgumentException::class.java)
+            .hasMessageContaining("지원하지")
```

Comment:
```suggestion
        assertThrows<IllegalArgumentException> { Operator.of(symbol) }
```
`assertThrows<>`를 사용하면, 예외 처리에 대해서도 간결하게 테스트 코드를 짤 수가 있습니다.

`assertThrows<>`와 `assertThatThrownBy`의 차이점은 무엇일까요? 🤔